src/hera/workflows/_mixins.py

```diff
@@ -615,23 +615,24 @@ class CallableTemplateMixin(ArgumentsMixin):
     not duplicated.
     """
 
+    directly_callable: bool = False
```

Comment:
I'm not totally sold on this as an interface name that conveys the right message (as a user: "callable templates are already callable, what does "directly" mean?") Thinking about something like `use_with_builder`, or `use_builder`. Naming is hard...

Reply:
>There are only two hard things in Computer Science: cache invalidation and naming things.
>
>-- Phil Karlton
I am not a big fan of `builder` because it's not exactly a builder either. Maybe somehow related to "first-call-the-function"? I am open to suggestions.

Reply:
More suggestions/ideas 😂
* `use_argument_kwargs`
* `use_function_kwargs`
* `use_inputs_as_kwargs` -> This then matches the idea for `Container`s?

Reply:
How about something like `function_params_only`? It is difficult to name 😢

Reply:
I think this is the clearest/most descriptive while being as succinct as possible 😂
```suggestion
    use_func_params_in_call: bool = False
```